src/tests/ftest/launch.py

```diff
@@ -1417,11 +1537,16 @@ def install_debuginfos():
         cmds.append(["sudo", "rm", "-f", path])
 
     if USE_DEBUGINFO_INSTALL:
-        yum_args = [
-            "--exclude", "ompi-debuginfo", "libpmemobj", "python", "openmpi3"]
+        distro_info = detect()
+        yum_args = ["--exclude", "ompi-debuginfo", "openmpi3"]
+        if "suse" in distro_info.name.lower():
+            yum_args.extend(["libpmemobj1", "python3"])
+        else:
```

Comment:
Should this branch only be run if the distro is actually EL with the `else:` falling through to an error about an unknown distribution?

Otherwise, you make the job of somebody porting this to a new distro much more difficult in having to hunt through code to find this out rather than being told clearly in an error message in the test output.

Reply:
we will fix this...